apps/console/src/features/identity-providers/meta/authenticator-meta.ts

```diff
@@ -63,7 +63,9 @@ export class AuthenticatorMeta {
             [ IdentityProviderManagementConstants.SAML_AUTHENTICATOR_ID ]: "Login users with " +
             "existing accounts in the Enterprise IdP.",
             [ IdentityProviderManagementConstants.EMAIL_OTP_AUTHENTICATOR_ID ]: "Two-factor authentication using " +
-            "one-time passcode sent via email."
+            "one-time passcode sent via email.",
+            [ IdentityProviderManagementConstants.SMS_OTP_AUTHENTICATOR_ID ]: "Two-factor authentication using " +
+            "SMS One Time passcode."
```

Comment:
Should we have `One Time` or `one-time` 🤔 I guess the below format is correct right?

```suggestion
            "SMS one-time passcode."
```

Reply:
Yes. That's the correct one. Will change (y)